v3/plugins/ms-vscode/protobuf/0.4.2/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: ms-vscode
```

Comment:
Publisher needs to match the publisher in the upstream repo's `package.json`, so this:

https://github.com/zxh0/vscode-proto3/blob/master/package.json#L6